plugins/plugins-api/pom.xml

```diff
@@ -4,7 +4,15 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
+  <dependencies>
```

Comment:
Could you put dependencies under parent. Not a huge deal but we should be consistent.